components/molecule/tabs/src/hoc/withStateActiveTab.js

```diff
@@ -24,7 +24,7 @@ const withStateActiveTab = BaseComponent => {
     }
 
     componentDidMount() {
-      const {children} = this.prop
+      const {children} = this.props // eslint-disable-line
```

Comment:
Why is that line disabled? Because PropTypes check? We might want to just add PropTypes instead disabling it... 🤔